models/stdp_connection.h

```diff
@@ -68,11 +68,12 @@
 
   FirstVersion: March 2006
   Author: Moritz Helias, Abigail Morrison
+  Adapted by: Philipp Weidel
   SeeAlso: synapsedict, tsodyks_synapse, static_synapse
 */
 
 // C++ includes:
-#include <cmath>
+#include <math.h>
```

Comment:
@weidel-p Yes, I know. But that information should be here in the C++ file, so that if someone reads this file five years from now will understand. E.g. `// C-header since copysign() is in C99 but not C++98`.